### ISSUE TITLE
Case Template: Custom Fields: need more customization

### ISSUE THREAD

**Issue by garanews** [TheHive-Project/TheHive#1001]
### Request Type
Feature Request

### Problem Description
1) need a possibility to set as mandatory a specific custom field
2) need a possibility to treats a custom field (with multiple choice) as a tag, let's call it "custom tag list"

Use case:

step 1)
![image](https://user-images.githubusercontent.com/16938405/58957748-8e655200-87a1-11e9-90ec-4a9eba8351fa.png)

step 2)
![image](https://user-images.githubusercontent.com/16938405/58957897-f5830680-87a1-11e9-94c2-a57135208628.png)

step 3)
![image](https://user-images.githubusercontent.com/16938405/58958296-e3559800-87a2-11e9-8c34-1293ad68a2c5.png)

**Comment by nadouani** [comment]
I guess the initial requirement is that you want the list of tags to allow a custom predefined list of tags right? Like in MISP?

**Comment by garanews** [comment]
Yes exacly, if not possible to use taglist coming from MISP (by your choice),  at least a field customizable. 
We're defining some mandatory tag that analyst has to set before sending data to MISP.
I tought modify the custom field was the easyest and fastest way to achieve it, but your suggestion is also good. :)

**Comment by nadouani** [comment]
I get it now, the mandatory thing is what made you think about the Custom Fields :)

**Comment by garanews** [comment]
Yes! 
We're playing with a data model where MISP events will need to have a minimum of mandatories tags (like ecsirt, dhs, ...) and due the analysts will publish MISP events through The Hive, a nice to have would be view these tag list already available in The Hive and set as mandatories.
